documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -255,11 +255,16 @@ You can specify which container image to use for a component using the `image` p
 * `Kafka.spec.entityOperator.topicOperator`
 * `Kafka.spec.entityOperator.userOperator`
 * `Kafka.spec.entityOperator.tlsSidecar`
+* `Kafka.spec.cruiseControl`
+* `Kafka.spec.kafkaExporter`
+* `Kafka.spec.kafkaBridge`
 * `KafkaConnect.spec`
 * `KafkaMirrorMaker.spec`
 * `KafkaMirrorMaker2.spec`
 * `KafkaBridge.spec`
 
+NOTE: Changing the Kafka image version does not automatically update the image versions for other Kafka components, such as Kafka Exporter.
```

Comment:
I would add that they are not version dependent and there is no need to configure them in any way.